ocp-config/prov-app/cm.yml

```diff
@@ -63,6 +63,9 @@ objects:
       bitbucket.default.user.group=${PROV_APP_CROWD_USER_GROUP}
       bitbucket.technical.user=${CD_USER_ID}
 
+      scm.global.readablerepos.opendevstack[0]=ods-jenkins-shared-library
+      scm.global.readablerepos.opendevstack[1]=ods-ods-quickstarters
```

Comment:
Typo? Should be just `ods-quickstarters`, right? Also, we need to read the MRO library. I'd rather work based on exclusion if possible?

Reply:
Done in #297